pyproject.toml

```diff
@@ -110,7 +110,7 @@ omit = [
 [tool.pytest.ini_options]
 addopts = "--doctest-modules"
 python_files = "test_*.py"
-testpaths = ["anndata", "docs/concatenation.rst"]
+testpaths = ["anndata/tests", "docs/concatenation.rst"]
```

Comment:
@flying-sheep, this disabled ~8 doctests

Reply:
From differences in `pytest --collect-only`

Reply:
Fixed in #1004.

What were the intended effects of this change?

Reply:
The VS Code test runner doesn’t work, but doing this right is of course more important.